runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -287,6 +305,30 @@ void registerTable(@Nonnull String namespace,
         }
     }
 
+    private void resetTrimmedTable(String fullyQualifiedTableName) {
```

Comment:
Can we add comments explaining why we are appending an empty checkpoint, and why it's fine if the checkpoint is appended but the transaction fails? Given that it's a very subtle and a little hacky solution.

Reply:
Done